dags/openshift_nightlies/scripts/install/cloud.sh

```diff
@@ -20,7 +20,6 @@ setup(){
     git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
     export PUBLIC_KEY=/home/airflow/workspace/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub
     export PRIVATE_KEY=/home/airflow/workspace/perf-dept/ssh_keys/id_rsa_pbench_ec2 
-    export ANSIBLE_FORCE_COLOR=true
```

Comment:
why removing this, AirFlow doesn't support it? curious.

Reply:
yeah airflow doesn't support the colored output and I think it might be the cause of fluentd rendering invalid characters in the logs (at least for the install task).

Reply:
+1 for response, -1 for airflow :laughing: